pkg/kubelet/cm/node_container_manager_linux.go

```diff
@@ -53,6 +53,33 @@ func (cm *containerManagerImpl) createNodeAllocatableCgroups() error {
 	return nil
 }
 
+func (cm *containerManagerImpl) updateInternalCapacity(rl v1.ResourceList) bool {
+	for k, v := range rl {
+		if _, ok := cm.internalCapacity[k]; !ok {
+			continue
+		}
+
+		if !v.Equal(cm.internalCapacity[k]) {
+			klog.Infof("node allocatable resource %q has changed", k)
+			cm.internalCapacity[k] = v
+			return true
```

Comment:
This will not work if multiple resource values have changed.

Would be nice with a test case handling it. :smile:

Reply:
Ouch, that was stupid indeed. Thanks for catching!